dotcom-rendering/index.d.ts

```diff
@@ -685,11 +685,6 @@ type SmallHeadlineSize =
 	| 'huge'
 	| 'ginormous';
 
-type AvatarType = {
-	src: string;
-	alt: string;
-};
-
```

Comment:
This type did not align with the wider usage of avatarUrl and there was no good solution for `alt` text. We now use the byline for alt text